llvm/test/CodeGen/X86/indirect-branch-tracking-jt.mir

```diff
@@ -0,0 +1,35 @@
+# RUN: llc %s -o - -mtriple=x86_64-- -run-pass=x86-ibt | FileCheck %s
```

Comment:
Check for both `cf-protection-branch` equals to 1 and 2? You can you sed to do so, e.g.,
```
# RUN: sed 's/LEVEL/1/g' %s | llc %s -o - -mtriple=x86_64-- -run-pass=x86-ibt | FileCheck %s --check-prefixes=L1
# RUN: sed 's/LEVEL/2/g' %s | llc %s -o - -mtriple=x86_64-- -run-pass=x86-ibt | FileCheck %s --check-prefixes=L2
...
  !0 = !{i32 1, !"cf-protection-branch", i32 LEVEL}
```

Reply:
The new test now checks for cf-protection-branch =  {0, 1, 2}.